core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -358,6 +360,24 @@ public FormValidation doCheckPollingThreadCount(@QueryParameter String value) {
                 return FormValidation.ok();
             return FormValidation.validateNonNegativeInteger(value);
         }
+
+        /**
+         * Performs syntax check.
+         */
+        public FormValidation doCheckScmpoll_spec(@QueryParameter String value,
+                                                  @QueryParameter boolean ignorePostCommitHooks,
+                                                  @AncestorInPath Item item) {
+            if (StringUtils.isBlank(value)) {
+                if (ignorePostCommitHooks) {
+                    return FormValidation.ok(Messages.SCMTrigger_no_schedules_no_hooks());
```

Comment:
Seems like this should be a warning—why would you even configure the trigger at all in this case?